### PR TITLE
feat(card): prefix change

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Prefix change in Card component

**Solving issue**  
Fixes: DTS-1629

**How to test**  
1. Go to the Storybook link below.
2. Open the Card component
3. It needs to be visually the same as the one in the main branch.
4. Prefix is removed from file names
5. sdds prefix is replaced with tds in the files

**Additional context**  
A small change was done to the story file in the slot area by adding an attribute slot to the icon component. Icon uses size prop instead of inline style.

### CODE REVIEW

File: core/src/components/card/card.tsx [hunk 1]
```diff
@@ -108,7 +109,7 @@ export class SddsCard {
             </div>
           </div>
         )}
-        {this.bodyDivider && <div class={`sdds-divider`}></div>}
+        {this.bodyDivider && <div class={`tds-divider`}></div>}
```
Comment (mJarsater) [inline core/src/components/card/card.tsx:112]: Could this use our own divider component instead? :)

File: core/src/components/card/card.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 @Component({
-  tag: 'sdds-card',
-  styleUrl: 'sdds-card.scss',
+  tag: 'tds-card',
+  styleUrl: 'card.scss',
   shadow: true,
 })
-export class SddsCard {
+export class Card {
```
[the remark this answers is not in the document]
Comment (theJohnnyMe) [reply core/src/components/card/card.tsx:8]: Damn! Thanks for checking!

**Review by mJarsater — APPROVED** [review]
Nice, nice, nice! One small question on the divider used in the card. 🚀

File: core/src/components/card/card.tsx (hunk 1 above)
Comment (theJohnnyMe) [reply core/src/components/card/card.tsx:112]: Not only that we could use it, but probably we need to use it as classes are no longer available. Great catch! I will inject the component, but most probably it won't be visible until the divider component is updated with a new prefix.

**Pull request merged**